lib/openxml/drawingml/properties/color_scheme.rb

```diff
@@ -0,0 +1,29 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class ColorScheme < OpenXml::Properties::ComplexProperty
+        include OpenXml::ContainsProperties
+        namespace :a
+        tag :clrScheme
+
+        attribute :scheme_name, displays_as: :name, expects: :string        # Required
+
+        property :dark_one, as: :color_scheme_dark_one                      # Required
+        property :light_one, as: :color_scheme_light_one                    # Required
+        property :dark_two, as: :color_scheme_dark_two                      # Required
+        property :light_two, as: :color_scheme_light_two                    # Required
+        property :accent_one, as: :color_scheme_accent_one                  # Required
+        property :accent_two, as: :color_scheme_accent_two                  # Required
+        property :accent_three, as: :color_scheme_accent_three              # Required
+        property :accent_four, as: :color_scheme_accent_four                # Required
+        property :accent_five, as: :color_scheme_accent_five                # Required
+        property :accent_six, as: :color_scheme_accent_six                  # Required
+        property :hyperlink, as: :color_scheme_hyperlink                    # Required
+        property :followed_hyperlink, as: :color_scheme_followed_hyperlink  # Required
```

Comment:
What does the `# Required` comment mean in this file? — didn't you add a `required: true` option to `property` (or was that `attribute`)?

Reply:
That was one of the ideas I was playing around with, but never implemented. In the meantime, I left the comments here to remind me or whoever might go back and implement such a thing down the line that these properties should receive the `required: true` (or equivalent option)

Reply:
What would happen if you just passed `required: true` to each of these properties _and it just didn't do anything_?

Would the `property` method raise an exception b/c it received an unknown option?

It'd be just like what you're doing, documenting the methods, but (obviously) in a way that we could work with programmatically — or just implement some day.

Reply:
Good question! I'll give it a try and see if I can change these comments into code.